src/main/java/frc/robot/subsystems/Hopper.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Hopper extends SubsystemBase {
+
+  TalonFX hopperMotor;
+
+  /** Creates a new hopper. */
+  public Hopper() {
+    hopperMotor = new TalonFX(0);
+
```

Comment:
No configure()? Just to set the motor to factory defaults (an unmodified TalonFXConfiguration)

Reply:
Good call out, we need to do this in all subsystems. I'll create an issue.